app/src/main/java/com/example/parkingsystem/util/ReservationCheck.java

```diff
@@ -0,0 +1,5 @@
+package com.example.parkingsystem.util;
+
+public enum ReservationCheck {
+    MISSING_DATE_START, MISSING_TIME_START, MISSING_DATE_END, MISSING_TIME_END, MISSING_PARKING_SPACE, MISSING_SECURITY_CODE, SUCCESS;
+}
```

Comment:
could you format this enum class like this:
public enum ReservationCheck {
    MISSING_DATE_START,
    MISSING_TIME_START,
    MISSING_DATE_END, ...

Reply:
Done!